pkg/utils/helm/helm.go

```diff
@@ -56,11 +56,9 @@ func GenerateValueFile(values interface{}) (valueFileName string, err error) {
 	return valueFileName, err
 }
 
-/**
-* generate helm template without tiller: helm template -f values.yaml chart_name
-* Exec /usr/local/bin/helm, [template -f /tmp/values313606961 --namespace default --name hj /charts/tf-horovod]
-* returns generated template file: templateFileName
- */
+//GenerateHelmTemplate generates helm template without tiller: helm template -f values.yaml chart_name
```

Comment:
Thanks for the suggestions from @xieydd . 
Please add space between '//' and content. Just like "// GenerateHelmTemplate"